pom.xml

```diff
@@ -30,18 +30,57 @@
         <!-- Remove when a new parent version with MTF is available -->
         <munit.input.directory>src/test/munit</munit.input.directory>
         <munit.output.directory>${basedir}/target/test-mule/munit</munit.output.directory>
-        <munit.extensions.maven.plugin.version>1.1.3</munit.extensions.maven.plugin.version>
-        <munit.version>2.3.15</munit.version>
-        <mtf.tools.version>1.1.2</mtf.tools.version>
+        <munit.extensions.maven.plugin.version>1.2.0-rc2</munit.extensions.maven.plugin.version>
+        <munit.version>3.1.0-rc3</munit.version>
+        <mtf.tools.version>1.1.2-rc1</mtf.tools.version>
         <mavenResourcesVersion>3.3.1</mavenResourcesVersion>
         <maven.surefire.plugin.version>2.22.2</maven.surefire.plugin.version>
-        <validations.module.version>1.4.6</validations.module.version>
-        <java.module.version>1.2.10</java.module.version>
-        <muleSdkApiVersion>0.3.0</muleSdkApiVersion>
+        <validations.module.version>2.0.6</validations.module.version>
+        <java.module.version>1.2.12</java.module.version>
+        <muleSdkApiVersion>0.6.0</muleSdkApiVersion>
+        <muleJavaEeBomVersion>4.6.0-20231025</muleJavaEeBomVersion>
```

Comment:
What is this used for ? Is this part of the java 17 changes is not is the original PR

Reply:
It's part of the original certification for java 17 that sabrina did. It's not in my original pr since it belongs to a previous PR that was already merged to main and I simply cherry picked it to include it in the java 17 release.